### ISSUE TITLE
Simple Example failing with Too Many Redirects

### ISSUE THREAD

**Issue by DougMay212** [spring-guides/tut-spring-boot-oauth2#76]
I have copied the Simple Example verbatum from this module, but it doesn't work. The browser (chrome) simply states that facebook tried to redirect me too many times. I clear my cache before attempting to access the local webpage. I'm not sure what I'm doing wrong, but as I've said, I copied the code from the Simple module word for word.

**Comment by dsyer** [comment]
You said you "copied the code". Did you try just git cloning it and running the actual code in the sample repo?

**Comment by DougMay212** [comment]
No, I tried generating the project from spring-io starter and following the README instructions. But when I couldn't get that working, I just copied and pasted each file, but without any luck. But I'll go ahead and close this - I followed the instructions for the github module and it's working like a charm.

**Comment by dsyer** [comment]
I guess that means there is something missing from the guide. Can you help find the difference?

**Comment by DougMay212** [comment]
I'm not sure, the only difference I could see was the spring actuator dependency not mentioned in the guide. But if it's working for you when you do a git clone of the repository, then I must have messed something up. I did use a properties file instead of the yaml file, so I could have gotten one of the properties wrong. I deleted my repo though as soon as I saw the github module worked. Sorry for the false alarm.
